packages/grafana-data/src/utils/dataFrameView.ts

```diff
@@ -68,4 +68,10 @@ export class DataFrameView<T = any> implements Vector<T> {
   toJSON(): T[] {
     return this.toArray();
   }
+
+  forEachRow<U>(iterator: (row: T) => void) {
```

Comment:
what is <U>?

Reply:
<U>

Reply:
Weird -- GitHub formatting does not like my U?

Reply:
Forgot to remove it, had it first as map function needing the generic param but realised forEach would be better fit for the usecase.